index.html

```diff
@@ -0,0 +1,535 @@
+                    <img src="./images/mobile_version/Card/Img_Plaholder.png" class="desktop-image-cover" alt="image">
+                    <div class="desktop-text-card-1">
+                        <h2 class="desktop-card-1-heading">Multi-Post Stories <br> Gain+Glory</h2>
+                    <ul class="desktop-list-languages">
+                        <li class="ruby">Ruby on rails</li>
+                        <li class="css">css</li>
+                        <li class="javascript">JavScript</li>
+                        <li class="html">html</li>
+                    </ul>
+                    <div class="desktop-button-frame">
+                        <button type="button" class="desktop-btn-green">See Project</button>
+                    </div>
+                    </div>
+                </div>
+
+
+                <div class="desktop-card-1">
+                    <img src="./images/mobile_version/Card/Img_Plaholder.png" class="desktop-image-cover" alt="image">
+                    <div class="desktop-text-card-1">
+                        <h2 class="desktop-card-1-heading">Multi-Post Stories <br> Gain+Glory</h2>
+                    <ul class="desktop-list-languages">
+                        <li class="ruby">Ruby on rails</li>
+                        <li class="css">css</li>
+                        <li class="javascript">JavScript</li>
+                        <li class="html">html</li>
+                    </ul>
+                    <div class="desktop-button-frame">
+                        <button type="button" class="desktop-btn-green">See Project</button>
+                    </div>
+                    </div>
+                </div>
+
+            </div>
+
+        <div class="desktop-card-work-row-2">
+          <div class="desktop-card-1">
+            <img src="./images/mobile_version/Card/Img_Plaholder.png" class="desktop-image-cover" alt="image">
+            <div class="desktop-text-card-2">
+                <h2 class="desktop-card-1-heading">Multi-Post Stories <br> Gain+Glory</h2>
+            <ul class="desktop-list-languages">
+                <li class="ruby">Ruby on rails</li>
+                <li class="css">css</li>
+                <li class="javascript">JavScript</li>
+                <li class="html">html</li>
+            </ul>
+            <div class="desktop-button-frame">
+                <button type="button" class="desktop-btn-green">See Project</button>
+            </div>
+            </div>
+           </div>
+
+           <div class="desktop-card-1">
+            <img src="./images/mobile_version/Card/Img_Plaholder.png" class="desktop-image-cover" alt="image">
+            <div class="desktop-text-card-2">
+                <h2 class="desktop-card-1-heading">Multi-Post Stories <br> Gain+Glory</h2>
+            <ul class="desktop-list-languages">
+                <li class="ruby">Ruby on rails</li>
+                <li class="css">css</li>
+                <li class="javascript">JavScript</li>
```

Comment:
- [ ] Looking a the Figma design, I think you need to adjust the styling of the project card. Currently, the language list is not properly aligned and the content tends to overflow. I'll suggest you look into this. :+1: :+1: 

- - Screenshot:
![Selection_083](https://github.com/siddghosh108/Portfolio-Desktop-Version/assets/107632780/ab0ef69d-431b-4db3-b315-5cd43577b706)

Reply:
![image](https://github.com/siddghosh108/Portfolio-Desktop-Version/assets/131663359/cc1a7a03-804b-45bf-a39b-0957d1ee01ec)
This issue is resolved now please see the image